### PR TITLE
Unclear statement that customer will confuse

### PR DESCRIPTION
**Pull request by SaiDuanZX**
Customer will keep trying to add the Visio and Project apps into WIP protection which seems we currently still does not support, we only support to add Visio and Project as Exempted apps at the moment

### CODE REVIEW

File: windows/security/information-protection/windows-information-protection/enlightened-microsoft-apps-and-wip.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Microsoft has made a concerted effort to enlighten several of our more popular a
 - Microsoft Remote Desktop
 
 >[!NOTE]
->Microsoft Visio and Microsoft Project are not enlightended apps and need to be exempted from WIP policy. If they are allowed, there is a risk of data loss. For example, if a device is workplace-joined and managed and the user leaves the company, metadata files that the apps rely on remain encrypted and the apps stop functioining.
+>Microsoft Visio and Microsoft Project are not enlightended apps and need to be exempted from WIP policy. They cannot be added to protected Apps for WIP at the moment. If they are allowed, there is a risk of data loss. For example, if a device is workplace-joined and managed and the user leaves the company, metadata files that the apps rely on remain encrypted and the apps stop functioining.
```
Comment (illfated) [inline windows/security/information-protection/windows-information-protection/enlightened-microsoft-apps-and-wip.md]: Typo corrections + please add MarkDown compatibility spacing:
```suggestion
> Microsoft Visio and Microsoft Project are not enlightened apps and need to be exempted from WIP policy. They cannot be added to protected Apps for WIP at the moment. If they are allowed, there is a risk of data loss. For example, if a device is workplace-joined and managed and the user leaves the company, metadata files that the apps rely on remain encrypted and the apps stop functioning.
```

File: windows/security/information-protection/windows-information-protection/enlightened-microsoft-apps-and-wip.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ Microsoft has made a concerted effort to enlighten several of our more popular a
 - Microsoft Remote Desktop
 
 >[!NOTE]
```
Comment (illfated) [inline windows/security/information-protection/windows-information-protection/enlightened-microsoft-apps-and-wip.md]: MarkDown compatibility spacing:
```suggestion
> [!NOTE]
```

**Comment by DulceMontemayor** [comment]
Please resolve branch conflicts. Thanks.

**Comment by e0i** [comment]
@SaiDuanZX 

Would you be able to resolve merge conflicts to proceed with the review? Thank you.

**Comment by illfated** [comment]
@e0i : it appears that the author of this PR (@SaiDuanZX ) does not have any visibly logged GitHub activity since the creation of this PR.

In case the author does not respond within a reasonable time limit, I suggest either asking one of the MS Docs members with write access to this PR to solve the merge conflict and then merge it, otherwise I offer to create a PR with @SaiDuanZX as the author of the file change in my PR, if that would be more practical.

**Pull request closed without merging**